### DISCUSSION TITLE
Setup 101 - Running on own desktop

### DISCUSSION THREAD

**Discussion by taz4535** [opendatacam/opendatacam#553] [Q&A]
Sorry, this might seem an immensely easy question, but for the life of me I'm struggling at every turn.

I have followed the setup in Azure for a deployed VM (no GPU, but stay with me) with Linux (Ubuntu 20.04), followed the setup and each step worked perfectly thru SSH, expect anything to do with GPU which I knew would fail as Azure doesn't have any GPU in AustraliaEast.

So, having a GTX 980 on my PC gives CUDA Enabled Compute Capability of 5.2. So hardware wise im good.
Docker Desktop installed.
Here's where I have tried a few different things. I've tried pulling ODC image straight into the docker (https://hub.docker.com/r/opendatacam/opendatacam obs using the desktop tag), and then using the terminal to run all the commands. Not successful, various errors.
Alternative was trying to emulate what I had done in Azure, so setup docker with Ubuntu docker and then run all the commands like I had previously, but meeting the same fate. It seems whenever I go thru docker I have to drop the sudo from the command lines, and then kinda all fails.

Looking for tips or tricks as to what im doing wrong. Should I be going thru a local VM like Hyper-V to get started?

**Comment by vsaw** [comment]
What kind of errors do you get? Can you provide a log?

**Reply by taz4535** [reply]
I just get the status 500. I'll grab the log as i'll reinstall it tonight.
![image](https://user-images.githubusercontent.com/30848350/178360779-6fd3973b-67a6-416e-8b19-e65634a928da.png)
